.github/workflows/release.yml

```diff
@@ -156,3 +156,19 @@ jobs:
         asset_path: pvsadm-binaries.tar.gz
         asset_name: pvsadm-binaries.tar.gz
         asset_content_type: application/tar+gzip
+
+    - name: Update Homebrew formula
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN}}
+        GH_TOKEN: ${{ secrets.GH_TOKEN}}
+      run: |
+        cd ci/
+        chmod +x brew_formula_updater.sh
+        git config --global user.name "pvsadm GitHub Actions"
+        git config --global user.email "pvsadmactions@github.com"
```

Comment:
who is the real owner of this token who is going to push the change? may that user name configuration is more relevant.

Reply:
The owner would be the same as to the `GITHUB_TOKEN` earlier set in the repo Action secrets. Since this automation is handled by Actions I have set the author as such to denote its a commit made by automated process.